python_modules/dagster/dagster/_utils/error.py

```diff
@@ -81,11 +89,47 @@ def _serializable_error_info_from_tb(tb: traceback.TracebackException) -> Serial
 ]
 
 
+def _should_redact_user_code_error() -> bool:
+    return os.getenv("DAGSTER_REDACT_USER_CODE_ERRORS") == "1"
```

Comment:
nit: might be good to be more permissive on value

Reply:
updated to take `1`, `true`, `t` (case insensitive)